tests/Resources/Page/OCPDashboard/OperatorHub/OperatorHub.robot

```diff
@@ -3,26 +3,31 @@ Library     SeleniumLibrary
 
 *** Keywords ***
 Install Operator
-    [Arguments]    ${operator}
+    [Arguments]    ${operator}      ${redhat_marketplace}=None
```

Comment:
why not True/False?

Reply:
True and False have special meaning in robot Framework. I try to avoid that. If you want me to change i can do. IMHO i will stick with None.

Reply:
mmmh I think if the parameter is a boolean, it would be better to limit its value range to 2 values. It's not a big deal to leave None